Index: kernel-source/drivers/staging/Kconfig
===================================================================
--- kernel-source.orig/drivers/staging/Kconfig
+++ kernel-source/drivers/staging/Kconfig
@@ -80,4 +80,6 @@ source "drivers/staging/qlge/Kconfig"
 
 source "drivers/staging/vme_user/Kconfig"
 
+source "drivers/staging/picocalc/Kconfig"
+
 endif # STAGING
Index: kernel-source/drivers/staging/Makefile
===================================================================
--- kernel-source.orig/drivers/staging/Makefile
+++ kernel-source/drivers/staging/Makefile
@@ -29,3 +29,4 @@ obj-$(CONFIG_PI433)		+= pi433/
 obj-$(CONFIG_XIL_AXIS_FIFO)	+= axis-fifo/
 obj-$(CONFIG_FIELDBUS_DEV)     += fieldbus/
 obj-$(CONFIG_QLGE)		+= qlge/
+obj-$(CONFIG_STAGING_PICOCALC)		+= picocalc/
